src/components/Checkbox/Checkbox.js

```diff
@@ -26,6 +26,13 @@ export default class Checkbox extends React.Component {
       }
     };
 
+    const info = this.props.info ?
+    <div className="chi-label__help">
+      <button className="chi-button -icon -sm -flat" id="example__help-button" aria-label="Help" data-target="#example__help-popover">
```

Comment:
What if this element is used twice on the canvas? Id duplication can break the functionality

Reply:
actually, why do we need the ID and data-target attributes if the designers are supposed to provide a custom interaction? And how do we trigger click, mouseDown, etc?

Reply:
added interactions for click, mouseOver and mouseLeave